electron/package.json

```diff
@@ -35,7 +35,7 @@
     "request": "2.85.0",
     "spellchecker": "3.4.4",
     "uuid": "3.2.1",
-    "winston": "https://github.com/wireapp/winston.git#2.2.0-e"
+    "winston": "https://github.com/wireapp/winston.git#2.2.0"
```

Comment:
That's interesting: https://github.com/wireapp/winston/releases

Most likely I had to do `2.2.0e` or something.. but I still think that `2.2.0-e` > `2.2.0`

Reply:
What if we want to increase the version but without changing the 2.2.0 (because it comes from upstream and we just have a patch..

Reply:
JS doesn't have a way to do that 😢. In Python you'd use a `+` separator to specify a "local version identifier" for exactly that use case, but that's not available here.

I'd recommend just closing this PR and letting Dependabot update you when a new version comes out. I'd also like to add functionality for Dependabot to detect that you're using a fork and create PRs to move you to the main library if/when the commit you're pinned on on that fork is included in a release, but it's one for the future, not now.

Reply:
Thanks @greysteil! Sometimes you are faster than bots :)